src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -9039,22 +9039,12 @@ public class MyAttribute : System.Attribute
 ";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
```

Comment:
>comp.VerifyDiagnostics( [](http://example.com/codeflow?start=12&length=23)

Consider adding `// PROTOTYPE: Should report CS0103: The name 'parameter' does not exist in the current context`.

Reply:
Will do.